evalml/tests/component_tests/test_delayed_features_transformer.py

```diff
@@ -193,49 +199,6 @@ def test_delayed_feature_extractor_maxdelay3_gap7(
     )
 
 
-@pytest.mark.parametrize("encode_X_as_str", [True, False])
```

Comment:
Since we can't initialize a numpy array as a categorical type without defeating the purpose of this test, I'm deleting it. I'm open to any recommendations for dynamically changing logical types within the `delayed_feature_transformer`.